pvlib/test/test_atmosphere.py

```diff
@@ -113,3 +111,82 @@ def test_first_solar_spectral_correction_supplied():
 def test_first_solar_spectral_correction_ambiguous():
     with pytest.raises(TypeError):
         atmosphere.first_solar_spectral_correction(1, 1)
+
+
+def test_kasten96_lt():
+    """Test Linke turbidity factor calculated from AOD, Pwat and AM"""
+    timestamps = pd.DatetimeIndex(MELBOURNE_FL[0])
+    timestamps = timestamps.tz_localize('UTC').tz_convert('Etc/GMT+5')
```

Comment:
one more that I just noticed... this line errors on the python27-min configuration because ``timestamps`` is already tz aware: https://travis-ci.org/pvlib/pvlib-python/jobs/198282511

I don't understand why the python-36 build completed, but we should still fix it.